src/lib/supabase/library/mockData.ts

```diff
@@ -0,0 +1,105 @@
+import {v4 as uuid} from 'uuid';
+import {
+  LibraryItem,
+  Hardware,
+  Book,
+  Film,
+  Controller,
+  Status,
+  Accessory,
+} from '~/models/models';
+
+const mockLibrary = [
+  {
+    // Hardware
+    inventory_id: uuid(),
+    created_at: new Date(),
+    updated_at: new Date(),
+    item_status: [Status.InStorage],
+    hardware_id: uuid(),
+    name: 'Digitakt',
+    quantity: 1,
+    image_url:
+      'https://cdn.mos.cms.futurecdn.net/aB3Zb7FFmvAzKnE25rPiYV-1200-80.jpg.webp',
+    tags: ['drum machine', 'synth'],
+    description:
+      'this is super cool sampler that work as a synth and drum machine. very power full',
+    accessories: null,
+    manufacturer: 'electron',
+    model: 'Digitakt',
+    weight: 1.5,
+  },
+  {
+    // Controller
+    inventory_id: uuid(),
+    created_at: new Date(),
+    updated_at: new Date(),
+    item_status: [Status.InStorage],
+    name: 'IK Multimedia iRig',
+    quantity: 1,
+    image_url:
+      'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSUm6pf5a6ovHD0bA2ghMhZ01Iw4oreF8sB514RWR1bI-9I0tvyhqFbY8kaKukrnmIToRg&usqp=CAU',
+    tags: ['controller', 'keyboard'],
+    description:
+      'IK Multimedia/s iRig Keys I/O combines a 25-key MIDI controller with an integrated audio interface to give you an all-in-one music production workstation that you can take anywhere. Compatible with iPhone, iPad, and Mac, it offers a full range of professional features, but keeps your setup clean and simple to help you focus on the music.',
+    accessories: null,
+  },
+  {
+    // Book
+    inventory_id: uuid(),
+    created_at: new Date(),
+    updated_at: new Date(),
+    item_status: [Status.InStorage],
+    book_id: uuid(),
+    name: 'The Synthesizer',
+    quantity: 1,
+    image_url:
+      'https://images-na.ssl-images-amazon.com/images/I/91E1s-Qq56L.jpg',
+    tags: ['book', 'paperback'],
+    description:
+      'The Roland Company, founded in 1972, was one of the earliest manufacturers of music synthesizers, and has shaped electronic music immeasurably.',
```

Comment:
If you need to make more mock data in the future, highly recommend looking into faker.js so you don't have to pull in hardcoded data / the data can be randomly generated and therefore can maybe catch more bugs.
https://www.npmjs.com/package/@faker-js/faker

Reply:
ill super down to check it out. would be awesome to not have to write stuff out.